src/main/java/com/activeviam/mac/cfg/impl/ManagerDescriptionConfig.java

```diff
@@ -672,9 +675,8 @@ private void chunkMeasures(final ICopperContext context) {
             .publish(context);
 
     Copper.measure(COMMITTED_ROWS_COUNT)
-        .divide(Copper.measure(CHUNK_SIZE_SUM))
-        .multiply(Copper.measure(DIRECT_MEMORY_SUM))
-        .withType(ILiteralType.LONG)
+        .multiply(directMemory)
+        .divide(chunkSize)
```

Comment:
The reason for doing first the division, then the multiply was to avoid overflows. Ideally, this operation should be perform with `double`s, then rounded to a long. I don't know if this is easily done in Copper.
If not, let's go with your solution. I let you decide. Ping me if you want my review again.

Reply:
Yes, you are completely right. That's why I was saying that doing this in Copper should be easier.

Reply:
I'm gonna check if there's an issue with Copper about that, because the multiplication between a double and a long should be a double without any rounding before the operation.